### PR TITLE
replaced oc rsh with oc exec bash -c 

### PR DESCRIPTION
**Pull request by izapolsk**
fix for #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -341,20 +341,20 @@ $ CONFIGMAP_GENERATOR_POD=`oc get pods | grep "httpd-configmap-generator" | cut
...
 ```
-$ oc rsh $CONFIGMAP_GENERATOR_POD httpd_configmap_generator ipa ...
+$ oc exec $CONFIGMAP_GENERATOR_POD  -- bash -c 'httpd_configmap_generator ipa ...
 ```
 
 Example configuration:
 
 ```
-$ oc rsh $CONFIGMAP_GENERATOR_POD httpd_configmap_generator ipa \
+$ oc rsh $CONFIGMAP_GENERATOR_POD -- bash -c 'httpd_configmap_generator ipa \
```
Comment (abellotti) [inline README.md]: oc exec ... ?

Comment (izapolsk) [reply README.md]: @abellotti, do you want me to replace rsh cmd with exec one ?
exec command  doesn't read bash config/profile by default and requires bash calls

Comment (jvlcek) [reply README.md]: 👍 LGTM @abellotti I believe this can and should be merged.

Comment (izapolsk) [reply README.md]: @jvlcek, done, thanks.

**Comment by abellotti** [comment]
Thanks @izapolsk for the update. LGTM!! 👍

**Pull request merged**